src/components/Modal/variant/SettingsModal.scss

```diff
@@ -35,4 +35,34 @@
 .settings-modal__show-author-checkbox {
   display: block;
   margin: 1.5em 0;
+}
+
+.settings-modal__delete-button {
+  border: 0;
+  background: none;
+  outline: none;
+  width: 100%;
+
+  font-size: $theme-font-size;
+  font-weight: bold;
+  text-transform: uppercase;
+
+  padding: $default-padding $large-padding;
+
+  margin-top: $large-margin;
+  color: $color-white;
+  background: $color-mint;
+
+  transition: background .2s ease-in-out, box-shadow .2s ease-in-out;
+
+  .delete-button__icon {
```

Comment:
The CSS guru Matthias Reuter taught me not to use nested selectors in SCSS, because it's easier to read when everything is defined in one line and also easier to `grep` / find those selectors. 
We can leave it here, as this version of the app will not be maintained in the future :)

Reply:
Okay